src/main/java/Tasks/Task.java

```diff
@@ -0,0 +1,58 @@
+package Tasks;
+
+public class Task{
+    protected String taskTitle;
+    protected boolean taskDone;
```

Comment:
Perhaps consider naming this boolean variable `isDone` instead, as according to the coding standard.